packages/webapp/pages/account/profile.tsx

```diff
@@ -34,6 +42,28 @@ const AccountProfilePage = (): ReactElement => {
     e.preventDefault();
     const values = formToJson<UpdateProfileParameters>(formRef.current);
 
+    if (values.username !== user.username) {
+      const isValid = handleRegex.test(values.username);
+
+      if (!isValid) {
+        displayToast('Invalid characters found in username!');
```

Comment:
Also maybe we can make this uniform somehow? (as there are multiple flows of setting/changing username)

Reply:
Made it a generic function now 🚢